### PR TITLE
Fix possible crash when clicking on Skeleton2D or Sprite editor options

### LINKED ISSUE #16891: "Sprite" button appears to any Node2D
**Godot version:**
08fbe0d

**Issue description:**
Likely caused by reduz's Mesh2D commit.
Clicking it makes the editor crash, unsurprisingly.

### PR DESCRIPTION
**Pull request by Chaosus**
Fixes #16891

### CODE REVIEW

**Comment by Noshyaar** [comment]
I'd prefer they're hidden than making an error message pops up when clicking on it.

**Comment by Chaosus** [comment]
@Noshyaar Ok, I hide these plugins by default, so they will likely be showed when appropriate sprites or skeletons is selected, and reduces the possibility of crash when checking for (!node)

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -5853,8 +5853,15 @@ EditorNode::EditorNode() {
...
 	add_editor_plugin(memnew(StyleBoxEditorPlugin(this)));
-	add_editor_plugin(memnew(SpriteEditorPlugin(this)));
-	add_editor_plugin(memnew(Skeleton2DEditorPlugin(this)));
+
+	// hide these plugins by default
+	SpriteEditorPlugin *sprite_editor_plugin = memnew(SpriteEditorPlugin(this));
+	Skeleton2DEditorPlugin *skeleton2d_editor_plugin = memnew(Skeleton2DEditorPlugin(this));
+	add_editor_plugin(sprite_editor_plugin);
+	add_editor_plugin(skeleton2d_editor_plugin);
+	sprite_editor_plugin->make_visible(false);
```
Comment (akien-mga) [inline editor/editor_node.cpp]: Maybe those two should be hidden directly in their constructor, as done for most other editor plugins?

**Comment by Chaosus** [comment]
Yeah, I rewrited it, thanks @akien-mga

**Pull request merged**